src/features/logging/aggregate/index.js

```diff
@@ -166,13 +168,18 @@ export class Aggregate extends AggregateBase {
       this.events.clear()
       this.events.clearSave()
     }
-    this.updateLoggingMode(LOGGING_MODE.OFF)
+    this.loggingMode = {
```

Comment:
Should this just call `this.updateLocalLoggingMode` to reduce the places that `this.loggingMode` is set?

Reply:
Could, but would have to change that to be an instance method though.